variantlib/models/base.py

```diff
@@ -14,3 +19,31 @@ def __post_init__(self) -> None:
             value = getattr(self, field_name)
             if (validator := field_def.metadata.get("validator", None)) is not None:
                 validator(value)
+
+
+@dataclass(frozen=True)
+class BaseHashableModel(ABC, BaseModel):
+    @abstractmethod
+    def _data_to_hash(self) -> list[bytes]:
+        """
+        Convert the object to a list of bytes that will be used for hashing.
+        """
+        raise NotImplementedError("Subclasses must implement this method.")
```

Comment:
For the record, I think it is customary to skip the message entirely, since it is never raised anyway.

Reply:
Oh right - I'll remove it ;)

Reply:
Fixed in 3d7d49b514d83f1c8a83734dfbba720b9393d1f5